dnf5/config/usr/share/dnf5/aliases.d/compatibility.conf

```diff
@@ -131,6 +131,24 @@ descr = "Alias for 'makecache'"
 group_id = 'commands-compatibility-aliases'
 complete = false
 
+['offline-distrosync']
+type = 'command'
+attached_command = 'distro-sync'
+group_id = 'commands-compatibility-aliases'
+complete = true
+attached_named_args = [
+  { id_path = 'distro-sync.offline' }
+]
+
+['offline-upgrade']
+type = 'command'
+attached_command = 'upgrade'
+group_id = 'commands-compatibility-aliases'
+complete = true
+attached_named_args = [
+  { id_path = 'upgrade.offline' }
+]
+
```

Comment:
I see these aliases don't match the dnf4 commands.
It basically changes `dnf offline-distrosync download` -> `dnf5 offline-distrosync` and discards the rest.

We do have all the functionality so I think we could build the aliases to (more or less) fit.

Something like: 
```
['offline-distrosync']
type = 'command'
attached_command = 'distro-sync'
group_id = 'commands-compatibility-aliases'
complete = true

['offline-distrosync.download']
type = 'command'
attached_command = 'distro-sync'
complete = true
attached_named_args = [
  { id_path = 'distro-sync.offline' }
]

['offline-distrosync.log']
type = 'command'
attached_command = 'offline.log'
complete = true
...
```

(If we decide to do that I would rather do it in a separate PR.)